<?xml version="1.0"?>

<launch>

      <node pkg="perception" type="detect_and_classify_signs.py" name="detect_and_classify_signs">
      <param name="model_path" value="$(find perception)/dd2419_detector_baseline/trained_models/det_2021-02-16_15-14-19-181578.pt"/>
      <param name="annotation_path" value="$(find perception)/dd2419_detector_baseline/dd2419_coco/annotations/training.json"/>  
      <param name="confidence_thres" value="0.5"/>
      </node>
        

</launch>